src/Kanbas/Courses/Quizzes/Preview/Question/index.tsx

```diff
@@ -1,7 +1,63 @@
-function QuizQuestionPreview() {
+import { useEffect, useRef } from "react";
+import { IKanbasQuizQuestion } from "../../../../store/interfaces/quizzes";
+import { FaFlag } from "react-icons/fa";
+import QuizQuestionHeader from "./Common/Header";
+import MultipleChoice from "./Common/MultipleChoice";
+import { eQuizQuestionType } from "../../../../store/enums/eQuizQuestionType";
+import ShortAnswers from "./Common/ShortAnswers";
+import "./index.css";
+
+function QuizQuestion({
+    question,
+    index,
+    flaggedQuestions,
+    handleFlagClick,
+    setRef }:
+    {
+        question: IKanbasQuizQuestion,
+        index: number,
+        flaggedQuestions: boolean[],
+        handleFlagClick: (index: number) => void,
+        setRef?: (index: number, ref: HTMLDivElement | null) => void
+    }
+) {
+
+    const ref = useRef<HTMLDivElement>(null);
```

Comment:
Why is this needed?

Reply:
Removing it now since moved the logic to redux state.